contents/ui/config.ui

```diff
@@ -1,66 +1,57 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
-	<class>Form</class>
-	<widget class="QWidget" name="Form">
-		<!-- After creating with Qt Designer, geometry was swapped out for minimumSize -->
-		<property name="minimumSize">
-			<size>
-				<height>142</height>
-			</size>
-		</property>
-		<property name="windowTitle">
-			<string>Form</string>
-		</property>
-		<widget class="QLabel" name="label_save_notice">
-			<property name="geometry">
-				<rect>
-					<x>0</x>
-					<y>10</y>
-					<width>541</width>
-					<height>41</height>
-				</rect>
-			</property>
-			<property name="font">
-				<font>
-					<pointsize>11</pointsize>
-					<weight>75</weight>
-					<bold>true</bold>
-				</font>
-			</property>
-			<property name="text">
-				<string>Due to a limitation of Kwin, you must disable and re-enable the script
-for changes to take effect.</string>
-			</property>
-		</widget>
-		<widget class="QLabel" name="label_blacklist">
-			<property name="geometry">
-				<rect>
-					<x>0</x>
-					<y>80</y>
-					<width>301</width>
-					<height>18</height>
-				</rect>
-			</property>
-			<property name="font">
-				<font>
-					<pointsize>11</pointsize>
-				</font>
-			</property>
-			<property name="text">
-				<string>Window Class Blacklist (comma-separated)</string>
-			</property>
-		</widget>
-		<widget class="QLineEdit" name="kcfg_blacklist">
-			<property name="geometry">
-				<rect>
-					<x>0</x>
-					<y>100</y>
-					<width>531</width>
-					<height>32</height>
-				</rect>
-			</property>
-		</widget>
-	</widget>
-	<resources/>
-	<connections/>
+ <class>Form</class>
+ <widget class="QWidget" name="Form">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>755</width>
+    <height>142</height>
+   </rect>
+  </property>
+  <property name="minimumSize">
+   <size>
+    <width>0</width>
+    <height>142</height>
+   </size>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QLabel" name="label_save_notice">
+     <property name="font">
+      <font>
+       <pointsize>11</pointsize>
+       <bold>true</bold>
+      </font>
+     </property>
+     <property name="text">
+      <string>Due to a limitation of Kwin, you must disable and re-enable the script for changes to take effect.</string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QLabel" name="label_blacklist">
+     <property name="text">
+      <string>Window Class Blacklist (comma-separated)</string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QLineEdit" name="kcfg_blacklist"/>
+   </item>
+   <item>
+    <widget class="QRadioButton" name="kcfg_shouldHideTiled">
```

Comment:
Since this is a boolean, should be a checkbox instead of a radio button. Would also be nice to have the option a little further down so it doesn't look like part of the blacklist, but that's not super important.

Reply:
Thanks! Fixed in [90aa5f3](https://github.com/fin444/truely-maximized/pull/6/commits/90aa5f3884cd9de92a0c0cff7199b9433017cc2e) and [a007d51](https://github.com/fin444/truely-maximized/pull/6/commits/a007d5115181d21d1249c68a7e3c3aeec5538a57). I added a horizontal line and a description label to separate the option from blacklist.